.ado/publish.yml

```diff
@@ -1,45 +1,193 @@
+                                        # The "manylinux" approach is used.
+        imageName: 'ubuntu-latest'      # Manylinux Py (x86_64) + VSCode Ext.
+        arch: x86_64
+      linux_ARM64:
+        imageName: 'ubuntu-latest'      # Manylinux Py (ARM64). Takes 45 - 65 minutes, hence is in a separate job.
+        arch: aarch64
+      mac:
+        imageName: 'macOS-latest'       # MacOS-specific Py (Mac is usually quite limited).
+        arch: x86_64
+      windows:
+        imageName: 'windows-latest'     # Win-specific Py + Platform-independent Py.
+        arch: x86_64
+  pool:
+    vmImage: $(imageName)
+  variables:
+    arch: $(arch)
+  timeoutInMinutes: 90
+
   steps:
   - task: UsePythonVersion@0
     inputs:
       versionSpec: '3.11'
 
   - script: |
       python ./prereqs.py --install
+    displayName: Install Prereqs
+
+  - script: |
       npm install -g @vscode/vsce
-    displayName: Install VSCE Pub Prereqs
-    
+    displayName: Install Prereqs for VSCode Extension
+    condition: and(eq(variables['Agent.OS'], '${{ variables.VSCodeExtensionBuildPlatform }}'), eq(variables['arch'], 'x86_64'))
+   
   - script: |
-      python build.py --wasm --npm --vscode
+      python build.py --wasm --npm --vscode --no-check --no-test --no-check-prereqs
     displayName: Build VSCode Extension
-      
+    condition: and(eq(variables['Agent.OS'], '${{ variables.VSCodeExtensionBuildPlatform }}'), eq(variables['arch'], 'x86_64'))
+
   - script: |
-      cp ../LICENSE.txt .
-      vsce package $(Ver.Major).$(Ver.Minor).$(Ver.Patch)
+      python -c "import shutil; shutil.copy2('../LICENSE.txt', '.')"
+      vsce package --pre-release
     displayName: Pack VSCode Extension
     workingDirectory: '$(System.DefaultWorkingDirectory)/vscode'
+    condition: and(eq(variables['Agent.OS'], '${{ variables.VSCodeExtensionBuildPlatform }}'), eq(variables['arch'], 'x86_64'))
+
+  - publish: $(System.DefaultWorkingDirectory)/vscode
+    artifact: VSIX
+    displayName: Upload VSCode Extension Artifact
+    condition: and(eq(variables['Agent.OS'], '${{ variables.VSCodeExtensionBuildPlatform }}'), eq(variables['arch'], 'x86_64'))
+
+  - script: |
+      python build.py --pip --no-check --no-test --no-check-prereqs
+    displayName: Build Platform-Dependent Py Packages, non-Linux
+    condition: ne(variables['Agent.OS'], 'Linux')
 
   - script: |
-      vsce publish --pre-release --pat $(PAT) 2>&1 > pub.log
+      set -x -e
+
+      export ARCH=${{ variables['arch'] }}
+
+      if [ "$ARCH" == "aarch64" ]; then
+        # The aarch64 (ARM64) emulation support:
+        sudo apt-get update
+        sudo apt-get install -y --no-install-recommends \
+          qemu qemu-system-misc qemu-user-static qemu-user
+      fi
+
+      chmod +x pip/build_manylinux.sh
+
+      docker run \
+        --env ARCH=$ARCH --rm -i -v `pwd`:/io \
+        quay.io/pypa/manylinux_2_28_$ARCH \
```

Comment:
I had to search around to find what this is. Maybe add a comment linking to `https://github.com/pypa/manylinux#docker-images` for the newbies like me.